src/main/java/org/apache/commons/configuration2/AbstractConfiguration.java

```diff
@@ -1517,6 +1547,25 @@ protected int sizeInternal() {
         return size;
     }
 
+    /**
+     * Checks if the specified value exists in the properties structure mapped by the provided keys.
+     *
+     * @param keys an Iterator of String keys to search for the value
+     * @param value the String value to search for in the properties
+     * @return true if the value is found in the properties, false otherwise
+     * @throws NullPointerException if keys or value is null
+     */
+    protected boolean contains(Iterator<String> keys, final String value) {
+        Objects.requireNonNull(keys, "Parameter \"keys\" cannot be null");
```

Comment:
See my previous comment RE `Objects.requireNonNull()`.
Same comment RE null checking in the first place. Why not return false?